dev/com.ibm.ws.rest.handler.validator/src/com/ibm/ws/rest/handler/validator/internal/ValidatorRESTHandler.java

```diff
@@ -308,4 +313,16 @@ private JSONObject toJSONObjectForThrowable(Map<String, ?> errorInfo, Throwable
 
         return json;
     }
+
+    @Trivial
+    private String resolvePotentialVariable(String value) {
+        if (value == null)
+            return value;
+
+        String resolvedVariable = variableRegistry.resolveString(value);
```

Comment:
You need to use `variableRegistry.resolveRawString(value)` so that the value is not path normalized.

Reply:
why don't we want path normalization here?

Reply:
Well for one, it's different behavior than we have for server config and datasource definition, so it may result in a different behavior than they'd actually get in their app. It also messes up some JDBC urls (especially for Oracle) among possibly other variables.